code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -261,19 +280,26 @@
 		return TRUE
 	return FALSE
 
-//Change organ status
+//Change limb between
+//Note:This proc only exists because I can't be arsed to remove it yet. Theres no real reason this should ever be used.
 /obj/item/bodypart/proc/change_bodypart_status(new_limb_status, heal_limb, change_icon_to_default)
-	status = new_limb_status
+	if(!(bodytype & new_limb_status) && (new_limb_status == BODYTYPE_ORGANIC))
+		bodytype = bodytype & ~BODYTYPE_ROBOTIC
+		bodytype = bodytype | BODYTYPE_ORGANIC
+	else
+		bodytype = bodytype & ~BODYTYPE_ORGANIC
+		bodytype = bodytype | BODYTYPE_ROBOTIC
```

Comment:
this seems weirdly hardcoded, why not use new_limb_staus that's here anyway?

Reply:
This is for legacy augment code support, I plan on removing it entirely in a future pr when i refactor augments.

Reply:
```suggestion
	if(!(bodytype & new_limb_status))
		bodytype &= ~(BODYTYPE_ROBOTIC | BODYTYPE_ORGANIC)
		bodytype |= new_limb_status
```
good enough for now but I hope you will follow up on this